engines/ospray/ispc/camera/OrthographicCamera.h

```diff
@@ -0,0 +1,70 @@
+// ======================================================================== //
+// Copyright 2009-2018 Intel Corporation                                    //
+//                                                                          //
+// Licensed under the Apache License, Version 2.0 (the "License");          //
+// you may not use this file except in compliance with the License.         //
+// You may obtain a copy of the License at                                  //
+//                                                                          //
+//     http://www.apache.org/licenses/LICENSE-2.0                           //
+//                                                                          //
+// Unless required by applicable law or agreed to in writing, software      //
+// distributed under the License is distributed on an "AS IS" BASIS,        //
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. //
+// See the License for the specific language governing permissions and      //
+// limitations under the License.                                           //
+// ======================================================================== //
+
+#pragma once
+
+#include "camera/Camera.h"
+
+namespace ospray
+{
+/*! \defgroup orthographic_camera The Orthographic Camera ("orthographic")
```

Comment:
remove this defgroup stuff is it clutters the doxygen output

Reply:
I reworked the whole doxygen comment a bit